### PR TITLE
:lipstick: no conference

### PR DESCRIPTION
**Pull request by guillaumecrespel**

### CODE REVIEW

File: src/components/marketing/logo/logo.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import xs from 'xstream'
 import { img } from '@cycle/dom'
 
 const view = () => xs.of(
-  img('.logo', { props: { src: '/logo-web2day.png' } }),
+  img('.logo', { props: { src: '/logo.png' } }),
```
Comment (fabienjuif) [inline src/components/marketing/logo/logo.js]: Where is the logo.png ?

**Review by fabienjuif — CHANGES_REQUESTED** [review]
Question

File: src/components/marketing/logo/logo.js (hunk 1 above)
Comment (guillaumecrespel) [reply src/components/marketing/logo/logo.js]: i wanted to put alakarte logo, waht do you think ?

Comment (fabienjuif) [reply src/components/marketing/logo/logo.js]: Yes go ahead

File: README.md [hunk 2]
```diff
@@ -8,11 +8,12 @@
...
+ - [BDX.io 2017](https://docs.google.com/presentation/d/1QipKxyUIcntcfenZPb6IzpMv11nKSCUyfVfB5VIgaWI/edit?usp=sharing)
 
 ## Live codings
- - [LC1 - First component](https://github.com/Trampss/cycle-music/pull/45)
- - [LC2 - Isolate components](https://github.com/Trampss/cycle-music/pull/42)
- - [LC3 - Map -xstream-](https://github.com/Trampss/cycle-music/pull/46)
- - [LC3bis - Filter -xstream-](https://github.com/Trampss/cycle-music/pull/47)
- - [LC4 - Merge -xstream-](https://github.com/Trampss/cycle-music/pull/43)
- - [LC5 - Driver](https://github.com/Trampss/cycle-music/pull/44)
+ - [LC1 - First component](https://github.com//cycle-music/pull/45)
```
Comment (fabienjuif) [inline README.md:14]: `alakarteio/`

File: public/global.css [hunk 3]
```diff
@@ -253,4 +270,4 @@ input[type=checkbox] {
 
 .wire > img {
     width: 3em;
-}
+}
```
Comment (fabienjuif) [inline public/global.css:273]: add empty line

**Comment by fabienjuif** [comment]
Why do you merge "rename DOM$ to DOM" here ?
You have to merge it to master because here we lose it.
Rebranding should be its own PR to I think

File: public/global.css (hunk 3 above)
Comment (guillaumecrespel) [reply public/global.css:273]: :ok_hand:

**Pull request closed without merging**